test/python/transpiler/test_basis_translator.py

```diff
@@ -1150,15 +1150,16 @@ def setUp(self):
         self.target.add_instruction(CXGate(), cx_props)
 
     def test_2q_with_non_global_1q(self):
-        """Test translation works with a 2q gate on an non-global 1q basis."""
+        """Test translation works with a 2q gate on a non-global 1q basis."""
         qc = QuantumCircuit(2)
         qc.cz(0, 1)
 
         bt_pass = BasisTranslator(std_eqlib, target_basis=None, target=self.target)
         output = bt_pass(qc)
-        # We need a second run of BasisTranslator to correct gates outside of
-        # the target basis. This is a known isssue, see:
-        #  https://docs.quantum.ibm.com/api/qiskit/release-notes/0.33#known-issues
```

Comment:
unrelated to the issue. Just too small to deserve its [own PR](https://github.com/Qiskit/qiskit/pull/11317)